examples/analytics.amp.html

```diff
@@ -127,6 +127,22 @@
 </script>
 </amp-analytics>
 
+<!-- Krux tracking. Ensure you replace CONFID with your configuration ID. -->
+<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/CONFID.json">
```

Comment:
the config does not exist. Is that intentional? Also is the name `confid.json` by design?